reddit/reaper.go

```diff
@@ -95,6 +98,25 @@ func (r *reaperImpl) sow(path string, values map[string]string) error {
 	return err
 }
 
+func (r *reaperImpl) get_sow(path string, values map[string]string) (Submission, error) {
+	r.rateBlock()
+	values["api_type"] = "json"
+	resp, err := r.cli.Do(
+		&http.Request{
+			Method: "POST",
+			Header: formEncoding,
+			Host:   r.hostname,
+			URL:    r.url(path, values),
+		},
+	)
+
+	if err != nil {
+		return Submission{}, err
+	}
```

Comment:
Would you also check here for non-200 return code? `err` may be nil even if Reddit decided this was an error. If you call `parse_submitted` with the response body, a 400 error for example would look like a json parsing error.

Reply:
Does `baseClient.Do` not already check for non-200 status codes [here](https://github.com/gastrodon/graw/blob/get-post-response/reddit/client.go#L41)?

Reply:
My mistake; I forgot that was `baseClient` and not `http.Client`.